framework/crud_resource.go

```diff
@@ -343,8 +343,3 @@ func (r *CRUDResource) EnsureDeleted(ctx context.Context, obj interface{}) error
 func (r *CRUDResource) Name() string {
 	return r.name
 }
-
-// TODO uncomment when Resource interface is redefined.
-//func (r *CRUDResource) Underlying() Resource {
-//	return r
-//}
```

Comment:
I did not follow that closely enough apparently. Does this mean we can remove the underlying methods from all of our resource implementations?

Reply:
Yes!